Bottom: b1ad32563972d70c7a93d5dfe1e6de9eee4512e0
Top:    729b947f8c52c96a11c4a33ff4ef447cd22cc5d1
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-04 14:35:54 -0700

Refresh of rs-maxseg

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 90502d3..e899e8a 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1818,8 +1818,7 @@ int rgetsockopt(int socket, int level, int optname,
 			*optlen = sizeof(int);
 			break;
 		case TCP_MAXSEG:
-			*((int *) optval) = (rs->cm_id && rs->cm_id->route &&
-					     rs->cm_id->route.num_paths) ?
+			*((int *) optval) = (rs->cm_id && rs->cm_id->route.num_paths) ?
 					    1 << (7 + rs->cm_id->route.path_rec->mtu) :
 					    2048;
 			*optlen = sizeof(int);
